src/hubble.rs

```diff
@@ -0,0 +1,250 @@
+use ethers::prelude::U256;
+use eyre::{bail, Result as EyreResult};
+use hyper::{body::Buf, client::HttpConnector, header, Client, Method, Request};
+use serde::{Deserialize, Serialize};
+use serde_json::{from_str, json, Value};
+use structopt::StructOpt;
+
+use crate::server::Error;
+
+#[derive(Clone, Debug, PartialEq, StructOpt)]
+pub struct Options {
+    #[structopt(long, env, default_value = "http://localhost:8080")]
+    commander_uri: hyper::Uri,
+
+    #[structopt(long, env, default_value = "50")]
+    airdrop_amount: u32,
+
+    #[structopt(long, env, default_value = "2")]
+    from_state_id: u32,
+}
+
+pub struct Hubble {
+    client:         Client<HttpConnector>,
```

Comment:
`reqwest` is the go-to http client library. It handles things like connection pooling that raw hyper doesn't. I think it also comes with build-in Serde support.